Template/Trigger/UserInteractionTrigger.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+namespace Piwik\Plugins\TagManager\Template\Trigger;
+
+use Piwik\Plugins\TagManager\Template\Trigger\BaseTrigger;
+
+class UserInteractionTrigger extends BaseTrigger
+{
+    public function getCategory()
+    {
+        return self::CATEGORY_USER_ENGAGEMENT;
+    }
+
+    public function getName()
+    {
+        return parent::getName();
+    }
+
+    public function getDescription()
+    {
+        return parent::getDescription();
+    }
+
+    public function getHelp()
+    {
+        return parent::getHelp();
```

Comment:
One other thought is that the help text should mention that this trigger only triggers once. Until I tested it and looked more closely at the code, I thought that it would trigger every time one of those events occurred instead of just the first time for any of them.

Reply:
Yes can be removed

Reply:
Thank you @JaJuMa . I think that new help text looks good and thank you for making the expected XML match. I think that this PR will be ready to merge once the [screenshots are added to git lfs](https://github.com/matomo-org/tag-manager/pull/582#issuecomment-1361389136).

Reply:
@AltamashShaikh @sgiehl @snake14 Screenshots are added to git lfs now.
Thanks again for all your great and speedy support getting this done!